tests/main/interfaces-kernel-module-load/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,21 @@
+name: test-snapd-sh
```

Comment:
nitpick but should this snap perhaps be named `test-snapd-kernel-module-load` instead? Especially since the cmd `bin/sh` doesn't appear to be used in the tests at all - is this necessary?

Reply:
Indeed, the test binary was a copy&pasted one :-) I've now renamed it and removed the unused program.